examples/1_pingpong.py

```diff
@@ -12,18 +12,19 @@
 import logging
 import os
 
-from pykeybasebot import Bot, ContentType
+import pykeybasebot.types.chat1 as chat1
+from pykeybasebot import Bot
 
 logging.basicConfig(level=logging.DEBUG)
 
 
 class Handler:
     async def __call__(self, bot, event):
-        if event.msg.content.type != ContentType.TEXT:
+        if event.msg.content.type_name != chat1.MessageTypeStrings.TEXT.value:
```

Comment:
i feel like no one is actually going to do this. they'll just write `"text"` here instead. 🙃